tests/functional/monitoring/prometheus/alerts/test_alert_mds_xattr_latency.py

```
@@ -220,32 +239,33 @@ def recover_mds_pods_if_not_running():
     for deployment in deployments_to_scale:
         deployment_name = deployment["metadata"]["name"]
         try:
             helpers.modify_deployment_replica_count(
                 deployment_name=deployment_name, replica_count=1
             )
-            log.info(f"Successfully scaled {deployment_name} to 1 replica")
-        except Exception as e:
-            log.error(f"Failed to scale MDS deployment {deployment_name}: {e}")
+            logger.info(f"Successfully scaled {deployment_name} to 1 replica")
+        except Exception:
+            logger.exception(f"Failed to scale MDS deployment {deployment_name}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)